### ISSUE TITLE
Cannot open Chinese file name

### ISSUE THREAD

**Issue by sword-huang** [markoteittinen/custom-maps#19]
When creating a new map, the file with Chinese file name cannot be opened, it will show that the file is too big.
If I change the file name to English, it will open normally.

**Comment by markoteittinen** [comment]
Yes, I made a little but embarrassing mistake when I changed the code to use the Android's new file storage access API. In version 1.8.1 the image files (jpg, png, and gif) fail to open if the file name contains spaces or special characters that are encoded in URLs.

Until the next version of Custom Maps is released, the workaround is to rename the image files to contain only letters A-Z, digits 0-9, and dashes and underscores (- and _). I have actually already fixed the issue in my own code.

But I cannot release a quick update to Custom Maps into Play Store, since the next release will no longer be able to access the already created map files in CustomMaps directory in the public internal storage of the devices. I need to give people time to get their maps migrated into the app's private internal storage which happens on the first launch after the app is updated to 1.8.1.
